db/migrate/20170124141701_add_uniq_index_on_minions_id.rb

```diff
@@ -0,0 +1,11 @@
+class AddUniqIndexOnMinionsId < ActiveRecord::Migration[5.0]
+  def up
+    remove_index :minions, :hostname
+    add_index :minions, :hostname, unique: true
+  end
+
+  def down
+    remove_index :minions, :hostname
+    add_index :minions, :hostname
+  end
+end
```

Comment:
Changing past migrations is bad because it will force everyone to rebuilt their database from scratch. Somebody might need their seed data for development and forcing them to drop everything is not the way to go. Of  course its just us for now but I prefer to do it the right way anyway.